cryptography/src/commonJvmAndroid/kotlin/com.wire.kalium.cryptography/ProteusClientCoreCryptoImpl.kt

```diff
@@ -42,6 +48,34 @@ class ProteusClientCoreCryptoImpl constructor(private val rootDir: String, priva
         }
     }
 
+    private fun cryptoBoxFilesExists(): Boolean =
+        CRYPTO_BOX_FILES.any {
+            File(rootDir).resolve(it).exists()
+        }
+
+    private fun deleteCryptoBoxFiles(): Boolean =
+        CRYPTO_BOX_FILES.fold(true) { acc, file ->
+            acc && File(rootDir).resolve(file).deleteRecursively()
+        }
+
+    private fun migrateFromCryptoBoxIfNecessary(coreCrypto: CoreCrypto) {
+        if (cryptoBoxFilesExists()) {
+            migrateFromCryptoBox(coreCrypto)
+        }
+    }
+
+    private fun migrateFromCryptoBox(coreCrypto: CoreCrypto) {
+        kaliumLogger.i("migrating from crypto box at: $rootDir")
+        coreCrypto.proteusCryptoboxMigrate(rootDir)
```

Comment:
We check if any of the Crytobox files exist, and we perform migration.
Does `CoreCrypto` handle the possibility of just some files exiting?
Shouldn't we play safe and check if `CRYPTO_BOX_FILES.all` exist?